taxonomy/definitions.md

```diff
@@ -0,0 +1,21 @@
+# Glossary of Terms: Understanding Lifecycle Phases in Technology
+
+This document aims to clearly define key terms related to the lifecycle of software and hardware products, providing a reference for understanding various phases from sales or creation to support termination. 
+
+**Note**: With open source software or hardware, there is typically no "end-of-sales".
+
+## 1. Software End-of-Life (SEoL) or Software End-of-Sales (SEoS)
+
+**Definition:** Software End-of-Life refers to the stage in a software product's lifecycle where it is no longer sold. 
+
+## 2. Hardware End-of-Life (HEoL) or Hardware End-of-Sales (HEoS)
+
+**Definition:** Hardware End-of-Life signifies the phase where a piece of hardware (like a server, router, or computer component) is no longer sold by its manufacturer.
+
+## 3. End-of-Software Maintenance or Security Vulnerability Support
+
+**Definition:** This term indicates the final phase of a software product’s lifecycle during which the developer ceases to provide maintenance services, including security patches, bug fixes, or any other form of software updates. Beyond this point, the software may become vulnerable to security risks as new threats emerge. However, a vendor may still support technical questions and may also investigate vulnerabilities in their products without providing a fix.
+
+## 4. Last Date of Support (End-of-Support)
```

Comment:
A comment was received to expand this title to "Last Date of Support ( LDoS ) or End-of-Support (EoS) Date"

Reply:
Agree, This titles are better however shorter form EoS can get confused between End of Sale & End of Support. If everyone agree we can use long form everywhere.

Reply:
I think abbreviations will occur naturally - better we define them here. So that it is clear, who gets the acronym "EoS"...